main.py

```diff
@@ -2,6 +2,47 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"),
+              ("Will", "Smith"), ("Jaden", "Smith"),
+              ("Scarlett", "Johansson")]
+
+    for item_first_name, item_last_name in actors:
+        Actor.objects.create(
+            first_name=item_first_name,
+            last_name=item_last_name
+        )
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(
+        name="Action"
+    ).delete()
+
+    Actor.objects.filter(
+        first_name="Scarlett",
```

Comment:
The filter condition `first_name="Scarlett"` will delete all actors with the first name 'Scarlett', including 'Scarlett Johansson'. If the intention is to delete only 'Scarlett Keegan', you should also filter by last name.